custom/yeksi_naa_reports/sqldata.py

```diff
@@ -188,34 +203,25 @@ def loc_name(self):
     @property
     def headers(self):
         if self.loc_id == 'district_id':
-            first_row = Translation.district[self.language()]
+            first_row = 'District'
         else:
-            first_row = Translation.region[self.language()]
+            first_row = 'Région'
 
         headers = DataTablesHeader(DataTablesColumn(first_row))
-        for month in self.months:
-            headers.add_column(DataTablesColumn(month.strftime("%B %Y")))
+        for month in self.month_headers():
+            headers.add_column(month)
         return headers
 
 
 class AvailabilityData(VisiteDeLOperateurDataSource):
-    slug = 'availability'
-    comment = 'Availability of the products at the PPS level: how many PPS had ALL products in stock'
-    title = 'Availability'
+    slug = 'disponibilité'
```

Comment:
Slugs are typically alphanumeric lowercase with dashes and underscores.  I'm not sure exactly where this slug is being used, but it may be safer to keep it in the ASCII character set.  Is it displayed to the user at all?